pom.xml

```diff
@@ -158,6 +160,7 @@
         <module>debezium-connector-sqlserver</module>
         <module>debezium-microbenchmark</module>
         <module>debezium-quarkus-outbox</module>
+        <module>test-openshift</module>
```

Comment:
I think you should add this to Travis condif file too - to the job that executes quarkus module to make sure that the module is buildable. Of course tests should not be executed :-)

Reply:
OK will have a look